susemanager-utils/susemanager-sls/susemanager-sls.changes.mc.enable-sl-micro-60

```diff
@@ -0,0 +1,2 @@
+- define bootstrap repo data for SL-Micro 6.0
+- calculate bootstrap repo path for SL-Micro 6
```

Comment:
Same, here, not sure about `SL-Micro` or `SL Micro` but there was mentioned somewhere `6.0`, but not `6`

Reply:
changed